descriptors/ModuleDescriptor-template.json

```diff
@@ -257,7 +257,8 @@
           ],
           "modulePermissions": [
```

Comment:
please add these new permissions to this list as well. Previously their endpoints were protected by other permissions covering multiple endpoints, but their endpoints are used.
"source-storage.records.matching.collection.post" 
"source-storage.records.formatted.item.get"
"source-storage.records.generation.item.put"